DQM/SiPixelPhase1Common/python/SpecificationBuilder_cfi.py

```diff
@@ -0,0 +1,116 @@
+import FWCore.ParameterSet.Config as cms
+
+# this is a pure Python module to build the deeply nested PSets that describe a 
+# SummationSpecification.
+# The C++ code assumes the form is fully correct, so you should always use this, 
+# which outputs a valid form.
+
+# these need to stay in sync with the C++ enums.  TODO: Can we use Python3 enum or sth.?
```

Comment:
Often people pass enums as strings from python to C++, and convert the strings to enums in the EDModule constructors (throwing exception if string is incorrect). That would also make the `edmConfigDump` (i.e. debugging) more readable than "magic constants".